app/controllers/assays_controller.rb

```diff
@@ -182,7 +185,9 @@ def propagate_permissions_to_children
 
       current_assay_policy = assay.policy
       # Clone the policy from the parent assay
-      assay.update(policy: @assay.policy.deep_copy)
+      assay_policy = @assay.policy.deep_copy
+      assay.update(policy: assay_policy)
+      assay.sample_type.update(policy: assay_policy) unless assay.sample_type.nil?
```

Comment:
bear in mind that this shares the policy between the assay and sample-type - so if the permissions for the assay change, so will with sample type. I can see why this may be intentional